lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm

```diff
@@ -194,79 +194,76 @@ sub execute {
             my $wgs_build = $clinseq_build->wgs_build;
 
             #WGS normal refalign build - source of normal WGS BAMs track
-            if ($wgs_build->can('normal_build')) {
-                my $wgs_normal_refalign_build = $wgs_build->normal_build;
-                my $wgs_normal_track = $self->generate_track_xml(
-                    '-build'         => $wgs_normal_refalign_build,
-                    '-resource_type' => 'bam'
-                );
-                my $wgs_normal_xml = $wgs_normal_track->{xml};
-                $wgs_normal_xml = $self->generate_panel_xml(
-                    '-track_xml'   => $wgs_normal_xml,
-                    '-panel_count' => $panel_count++
-                );
-                push(@resources, $wgs_normal_track->{resource});
-                $main_features_track_xml .= $wgs_normal_xml;
-            }
+            my $wgs_normal_build = $wgs_build->can('normal_build') ? $wgs_build->normal_build : $wgs_build;
```

Comment:
Would this be something useful to add to the `Genome::Model::Build::SomaticInterface`? Something like `build_for_normal_bam`?

(I was surprised to see that `tumor_bam` and `normal_bam` aren't really a part of that interface.)